### DISCUSSION TITLE
家人们VSCode怎么提交到已开启的PR

### DISCUSSION THREAD

**Discussion by MCTF-Alpha-27** [CFPAOrg/Minecraft-Mod-Language-Package#2041] [Q&A] [answered]
之前不会搞，手动提交了一次，后面想用VSCode的源代码管理来提交好像得重新开一个PR才行了

**Comment by Cyl18** [comment]
如果你用的是 GitHub Desktop，可以切换分支；

VSCode 的话：
首先 clone 的一定是你自己 fork 的库；
然后在VSCode -> Source Control -> Remotes -> origin -> 找到PR对应的分支 -> Switch To Branch

**Comment by n0099** [comment, marked as answer]
把pr页面的权威url的.com tld域改成.dev即可
比如`https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/2034`
就是https://github.dev/CFPAOrg/Minecraft-Mod-Language-Package/pull/2034
